test/e2e/lib/pages/external/jetpackcom-pricing-page.js

```diff
@@ -12,13 +12,14 @@ import * as driverHelper from '../../driver-helper';
 export default class JetpackComPricingPage extends AsyncBaseContainer {
 	constructor( driver, url ) {
 		if ( ! url ) {
-			url = 'https://jetpack.com/pricing/';
+			url = 'https://cloud.jetpack.com/pricing/';
 		}
-		super( driver, By.css( '.plans-pricing-comparison' ), url );
+		super( driver, By.css( '.is-section-jetpack-cloud-pricing .selector-alt__main' ), url );
 	}
 
-	async buyPremium() {
-		const buyPremiumSelector = By.css( 'a[href*="premium"]' );
-		return await driverHelper.clickWhenClickable( this.driver, buyPremiumSelector );
+	// To buy a Jetpack Plan, we search for `planName` in every CTA on the page.
+	async buyJetpackPlan( planName ) {
+		const planCTA = By.xpath( `//button[@type='button' and contains(., 'Get ${ planName }')]` );
```

Comment:
Generally, test locators are not recommended, since the text could change more often, which could lead to test breakage more often. Also, they might not work in a non-english locale.

Can we, instead of using text locators, add data attributes to the components we are interested in?

I see similar text locators in few other places, so it might make sense to add data attributes there as well.

Reply:
Since this page will probably be changed a lot in the next coming weeks, I think it will be better if we use data-attributes. They will also serve as warning for developers that they might break tests if they don't preserve them.